recipe/bld.bat

```diff
@@ -11,12 +10,22 @@ set CMAKE_CONFIG="Release"
 
 dir /p %LIBRARY_PREFIX%\lib
 
+echo "Constructing makefiles"
 cmake -LAH -G"NMake Makefiles"                               ^
     -DCMAKE_BUILD_TYPE=%CMAKE_CONFIG%                        ^
     -DCMAKE_FIND_ROOT_PATH="%LIBRARY_PREFIX%"                ^
     -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ..
```

Comment:
I think you just need to add that options which I told you and it will build
```
    -DCMAKE_PREFIX_PATH="%PREFIX%"                           ^
    -DCMAKE_CXX_STANDARD:STRING=17                           ^
    -DCMake_HAVE_CXX_MAKE_UNIQUE:INTERNAL=TRUE               ^
```